Makefile

```diff
@@ -278,6 +278,12 @@ react-build:
 	rsync -av --delete --delete-excluded --exclude=reports \
 		frontend/build/ lib/streamlit/static/
 
+.PHONY: frontend-fast
+frontend-fast:
```

Comment:
Oh, and can you add a comment (between the PHONY line and the `frontend-fast:` line) explaining what this does differently from `make frontend`?

Reply:
Sure 👍 comment `# Build frontend into static files faster by setting BUILD_AS_FAST_AS_POSSIBLE=true flag, which disables eslint and typechecking.` is there 👍